### ISSUE TITLE
A question about parameter discretization and normalization

### ISSUE THREAD

**Issue by Ullar-Kask** [tensorflow/recommenders#285]
Hi,

I have a question regarding discretization and normalization. In the "Building deep retrieval models" example timestamp values are both discretized into buckets:

`
self.timestamp_embedding = tf.keras.Sequential([
   tf.keras.layers.experimental.preprocessing.Discretization(timestamp_buckets.tolist()),
   tf.keras.layers.Embedding(len(timestamp_buckets) + 1, 32),
])
`

and normalized:

`
self.normalized_timestamp = tf.keras.layers.experimental.preprocessing.Normalization()
`

Layers are called as:

`
tf.concat([
        self.timestamp_embedding(inputs["timestamp"]),
        self.normalized_timestamp(inputs["timestamp"]),
], axis=1)
`

What is the reasoning behind feeding timestamp values twice into the network? Other parameters are fed once.

TIA,

Üllar Kask

**Comment by Ullar-Kask** [comment]
Thanks a lot!
